pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+            page.raise_for_status()  # Raises an HTTPError if the status is 4xx, 5xx
+
+            soup = BeautifulSoup(page.content, 'html.parser')
+            h3_list = soup.select('h3')
+
+            if h3_list:
+                h3_text = h3_list[0].get_text().split(" - ")[0]
+                return remove_prefix(h3_text, "https://escapefromtarkov.gamepedia.com/")
+            else:
+                return None
+
+        except Exception as e:
```

Ruff (0.8.0)

Do not catch blind exception: `Exception`

(BLE001)